netfns/netfns.go

```diff
@@ -275,3 +276,34 @@ func CollectTAPFunctions(devices []string) ([]ghw.PCIAddress, error) {
 	}
 	return addresses, nil
 }
+
+func CollectVirtualFunctionsOnHardCodedPF(pfDevice string, numVFs int, offset int) ([]ghw.PCIAddress, error) {
```

Comment:
Looks good to me. But this function name seems too long. "Collect" gives the reader the impression that something is grabbed from the system, but actually it is doing string magics. What about just something like `GenerateVFNames`?

Reply:
Yeah, I was trying to adhere to the rest of the naming in the file but "collect" doesnt fit really. I liked the idea of using "Generate" instead.
As the other function names are not using shortened form for Virtual Function, I would still keep that but shorten it like this: `GenerateVirtualFunctions`